src/pushsource/_impl/model/ami.py

```diff
@@ -92,6 +92,27 @@ def _from_data(cls, data):
         return cls(**kwargs)
 
 
+@attr.s()
+class AmiAccessEndpointUrl(object):
+    """Access Endpoint URL to be associated with a Marketplace VM."""
+
+    port = attr.ib(type=int, validator=instance_of(int))
+    """Port to access the endpoint URL."""
+
+    protocol = attr.ib(type=str, validator=instance_of_str)
+    """Protocol to access the endpoint URL (http, https)."""
+
+    @classmethod
+    def _from_data(cls, data):
+        """Instantiate SecurityGroup from raw dict"""
```

Comment:
False copy-pasted comment, this isn't a SecurityGroup.

Deleting a comment entirely is preferred over copy-pasting an incorrect one.